### DISCUSSION TITLE
PrimeNG filter menu -> Remove Rule -> Change focus to Apply button instead of Clear buttons

### DISCUSSION THREAD

**Discussion by gopikrishna-manifest** [primefaces/community#1719] [PrimeNG]
When we click on "Remove Rule" button in PrimeNG filter menu, the focus is going to **Clear** button and when user click on **Enter** button it is clearing the remaining rules. Is there any way to change the focus to **Apply** button instead of **Clear** button?

**Steps**
1) Open the Primeng filter menu example. https://primeng.org/table#filter-menu
2) Click on Filter icon beside Name / Country column and open the filter menu.
3) Enter some text in the text box. 
![image](https://github.com/primefaces/community/assets/85510959/e77f8e5f-0922-4aa8-a60b-b54792c0e52f)
4) Click on "Add Rule" button
![image](https://github.com/primefaces/community/assets/85510959/c5036455-abb4-4538-b48d-62922aa8a009)
5) Click on "Remove Rule" button in the new Rule block
6) Now press "Enter" button
7) You can see the focus is on "Clear" button now and the text in the first Filter rule is cleared.
![image](https://github.com/primefaces/community/assets/85510959/e9b76265-427e-4a2d-9f13-52a7862c6f8b)

As per the PrimeNG code I found that the default focus is changing to "Close" button. Is it possible to change it to "Apply" button instead of "Clear" button?

**Comment by cetincakiroglu** [comment]
Hi,

It's made by design, we'll wait for the community upvotes to change it.
